src/utils/documentSource.ts

```diff
@@ -0,0 +1,5 @@
+export enum DocumentSource {
+  DOCUMENT_SOURCE_TREEVIEW = 'treeview',
+  DOCUMENT_SOURCE_PLAYGROUND = 'playground',
+  DOCUMENT_SOURCE_COLLECTIONVIEW = 'collectionview'
```

Comment:
Yes, we use it for telemetry, but in order to make it work, we need to pass it to several places in code. And I found it weird to require a telemetry model in types.ts to import this enum therefore moved it to a separate file. I also have noticed that we have types in type.js, but also separate files with types such as connectionModelType.ts or dataServiceType.ts. I think it would be nice to place all of them together. Eiter all in a single types file or each type into a separate file and place, all types file in a single types folder. But I think to do it after discussion with you as a small separate PR. What do you think?

Reply:
sure sounds good